docs/client/initializing.md

```diff
@@ -0,0 +1,105 @@
+# <img align="center" src="../images/logo.png">  Initializing Your Python Client
+
+The first step to using your generated client in code is to import and initialize your client. Our SDKs are modelled such
+that the client is the main point of access to the generated code.
+
+## Importing Your Client
+
+You import your client from the namespace specified when generating (under flag `--namespace`). For the sake of this example,
+let's say the namespace is `azure.pets`. Your client's name is detailed in the swagger, (TODO link to swagger docs), and let's say
+ours is called `PetsClient`.
+
+Putting this together, we import our client like so:
+
+```python
+from azure.pets import PetsClient
+```
+
+### Dependencies of Your Client
+
+The only scenario the generated code can force dependencies is if you generate with a `setup.py` file using the `--basic-setup-py` flag.
+The following are core libraries your generated code depend on, and the minimum version we highly recommend:
+
+| Library | Description | Min Version
+|------------------|-------------|-------------
+|[`azure-core`][azure_core_library]|The most important library to have installed. It provides shared exceptions and modules for all the Python SDK client libraries.|1.8.2
+|[`msrest`][msrest_library]|Library mainly used for serializing and deserializing objects now|0.6.18
+|[`azure-mgmt-core`][azure_mgmt_core_library]|Required if you're generating mgmt plane code (see `--azure-arm` flag in our [flag index][flag_index]. Provides mgmt plane specific shared exceptions and modules.|1.2.1
+
+> Note: We highly recommend tying your library to a major version, for instance, adding `azure-core<2.0.0` to tie the `azure-core` library to `1.x.x`
+
+## Initializing Your Client
+
+Next, on to initialization. Your constructor can take any number of parameters. If your client has no parameters (no client parameters detailed
+in the swagger (TODO: link to swagger docs) and you choose to generate without credentials), initializing your client would just look like
+
+```python
+from azure.pets import PetsClient
+
+client = PetsClient()
+```
+
+However, by default we generate clients with credentials, so continue on to [Authenticating Your Client](#authenticating-your-client "Authenticating Your Client")
```

Comment:
yeah that's def fair. I had the other as the first one since it's the simplest. The more I think about it though, I think I can get rid. of that section and meld these 2 together. thanks!